components/connectivity-adapter/internal/appregistry/service/handler.go

```diff
@@ -159,7 +161,31 @@ func (h *Handler) Get(writer http.ResponseWriter, request *http.Request) {
 
 func (h *Handler) List(writer http.ResponseWriter, request *http.Request) {
 	h.logger.Println("List")
-	// TODO: Implement it
+	//TODO: Implement it, currently this endpoint purpose is for manually testing appdetails middleware
+
+	ctx := request.Context()
+	app, err := appdetails.LoadFromContext(ctx)
+	if err != nil {
+		wrappedErr := errors.Wrap(err, "while getting service from context")
+		h.logger.Error(wrappedErr)
+		reqerror.WriteError(writer, wrappedErr, apperrors.CodeInternal)
+		return
+	}
+
```

Comment:
this method is too long - it will be removed soon, just log application from context.